bcipy/signal/model/offline_analysis.py

```diff
@@ -333,115 +338,125 @@ def analyze_gaze(
         labels = np.array([sym] * len(le))  # Labels are the same for both eyes
         train_le, test_le, train_labels_le, test_labels_le = subset_data(le, labels, test_size=0.2, swap_axes=False)
         train_re, test_re, train_labels_re, test_labels_re = subset_data(re, labels, test_size=0.2, swap_axes=False)
+        train_dict[sym] = np.concatenate((train_le, train_re), axis=0)
         test_dict[sym] = np.concatenate((test_le, test_re), axis=0)
 
         # Fit the model based on model type.
-        # Model 1: Fit Gaussian mixture (comp=2) on each symbol and each eye separately
         if model_type == "Individual":
-            model.fit(train_re)
+            # Model 1: Fit Gaussian mixture on each symbol separately
+            model.fit(train_dict[sym])
 
-            means, covs = model.evaluate(test_re)
+            means, covs = model.evaluate()
 
             # Visualize the results:
             figure_handles = visualize_gaze_inquiries(
                 le, re,
                 means, covs,
                 save_path=None,
-                show=show_figures,
+                show=False,
```

Comment:
Why have this call if no saving or showing happens?

Reply:
`visualize_gaze_inquiries` is called for each symbol in this specific loop for debugging purposes. You're right that we don't save or show them during a regular offline analysis, instead we show all symbols together at the end. So I will remove this loop.